xds/internal/testutils/fakeclient/client.go

```diff
@@ -49,6 +50,8 @@ type Client struct {
 	rdsCb  func(xdsclient.RouteConfigUpdate, error)
 	cdsCbs map[string]func(xdsclient.ClusterUpdate, error)
 	edsCb  func(xdsclient.EndpointsUpdate, error)
+
+	Closed *grpcsync.Event // fired when Close is called.
```

Comment:
Delete `closeCh` in line 45

Reply:
Whoops, lost that part of the cleanup from before.  Fixed, thanks.